notificator/notificator_queue.py

```diff
@@ -213,12 +217,14 @@ async def _process_normal_queue(self):
                 del self.notif_cves_queue[sys_vuln_id]
                 continue
 
+            inventory_id = sys_vuln_row[6]
+            display_name = sys_vuln_row[7]
```

Comment:
Hmm :thinking:  I wonder how it could be easier if one would have invented named columns. :thinking: Like a `RealDictCursor`.

Reply:
Yeah that is cool but we are using `asyncpg` not psycopg :D